frontend/src/container/TopNav/DateTimeSelection/index.tsx

```diff
@@ -187,6 +190,11 @@ function DateTimeSelection({
 
 		const { maxTime, minTime } = GetMinMax(value, getTime());
 
+		urlQuery.set(QueryParams.startTime, minTime.toString());
+		urlQuery.set(QueryParams.endTime, maxTime.toString());
+		const generatedUrl = `${location.pathname}?${urlQuery.toString()}`;
+		history.replace(generatedUrl);
+
 		initQueryBuilderData(updateStepInterval(stagedQuery, maxTime, minTime));
```

Comment:
`initQueryBuilderData` is also replacing the history can we optimise this one ?

Reply:
let me check this in follow up PR

Reply:
if you are working an issue which is intermediately fixed create a sub task which is not closing the issue otherwise we will lose the track

Reply:
Yes we have one other issue mentioned in the same ticket so will resolve with the same